Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Menopause and Andropause

## Changelog:
- Added On Menopause for females and On Andropause for males it appears if:
  - The character is a long-lived race and has 5 children for females and 7 children for males.
  - The character has reached the age of 45 for females and 60 for males (long-lived races have scaled equivalents.


# How to test:
Observe the game. Check if the trait is added right and the AI doesn't marry infertile characters.
